src/models/view/cardView.ts

```diff
@@ -18,6 +18,10 @@ export class CardView implements View {
         // ctor
     }
 
+    get maskedCode(): string {
+        return this.code != null ? '••••' : null;
```

Comment:
Should we make the number of dots equal to `this.code.length`? On password we purposely don't do that to mask the length as well, but I think it's probably ok here since it's probably either 3 or 4.

Reply:
Yup - updated that to show same number of dots, whatever the length.